### PR TITLE
strcloud.in - popups

### PR DESCRIPTION
**Pull request by wout-codes**
`https://strcloud.in/v/A0jRRKRgxgHmxo`

### CODE REVIEW

**Review by zloyden — CHANGES_REQUESTED** [review]
@wout-codes also add `strcloud.link`.

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -243,6 +243,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! strcloud.in - popups
+strcloud.in#%#//scriptlet("prevent-window-open")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:247]: Do you need this one? This `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")` is enough on my end.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -1223,6 +1223,7 @@ procrackerz.org,chapterupdates.com,babelnovel.com,katzen-fieber.de,xn--80aaxitdb
 !+ NOT_PLATFORM(windows, mac, android, ext_ff)
 quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !--- JS ----
+streamtape.com,strcloud.in#%#//scriptlet("prevent-addEventListener", "contextmenu", "event.dispatch.apply")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1226]: What is this rule for?

Comment (wout-codes) [reply AnnoyancesFilter/sections/tweaks.txt:1226]: To fix Anti-right click.

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:247]: I will remove it.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:247]: @zloyden 
Btw, the normal HTML Rule for this scriptlet (`#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")`) isn't working with the `min-length` and `max-length`. If I remove it and craft it to something like this  `strcloud.*$$script[wildcard="*function*break;case*"]`, then it's working.

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt]: I have the same. Please check `strcloud.*$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="100000"]`. The script is deleted normally.

File: AnnoyancesFilter/sections/tweaks.txt (hunk 2 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt]: Finally reproduced it. Thanks.

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt]: @zloyden Should I override the current rule or add it under HTML?

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt]: You can rewrite the rule for the pull request - it's ok here.
Please notice that you should change it in 2 places.
1. Comment section.
2. Actual rule.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt]: Changed.

**Review by zloyden — APPROVED** [review]

**Pull request merged**